### PR TITLE
Updated roadmap doc and the README links to it.

### PR DESCRIPTION
**Pull request by ElleEss**
#### Description
Roadmap docs updates:
1. Added title and date.
2. Added definition and date to 'first look'.
3. Added link to feedback information.
4. Renamed to standard lower-case docs naming convention.
5. Did not update roadmap text (despite what the diffs say).

README updates
1. Set a link to the roadmap from the docs readme.
2. Updated the text and link to the roadmap from the repo readme.
#### Tests
Clicked on the links
#### Documentation
Is documentation.
#### Primary reviewers
@ernopp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ For prerequisites, installation and setup, see the [Installation and setup](docs
 Currently the Unity GDK is in its early development, so there is no migration path from the [Unity SDK](https://github.com/spatialos/UnitySDK) to the Unity GDK. There will be a migration path in later releases.
 
 ## Roadmap
-See the latest Roadmap [here](docs/Roadmap.md) and don't hesitate to feedback!
+See the [latest roadmap](docs/roadmap.md) and let us know what you'd like to see in there.
```
Comment (anne-edwards) [inline README.md]: The link doesn't work for me unless I change `roadmap` to `Roadmap` - do you need to rename `docs/Roadmap.md`?

File: docs/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ We recommend you start by following the [Installation and setup guide](setup-and
 
 Note that the Unity GDK documentation assumes that you are familiar with SpatialOS concepts and the Unity ECS.
 
-We want your feedback on the Unity GDK and its documentation - see [Give us feedback](../README.md#give-us-feedback).
+We want your feedback on the Unity GDK, its documentation and its [roadmap](roadmap.md) - see [Give us feedback](../README.md#give-us-feedback).
```
Comment (anne-edwards) [inline docs/README.md]: As above re. `roadmap`/`Roadmap`

File: docs/Roadmap.md [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
+# SpatialOS Unity GDK roadmap
+**Date:** 2018-06-19
+
+We'd love to hear what you think about the roadmap; [give us feedback](../README.MD#give-us-feedback) on what you'd like to see in it.
 
 | | Faster development iteration | Native workflows| High performance|
 |------|---|----|--|
-|First look|**Build less often**<br/>Run client and server simultaneously in your Unity Editor.<br/><br/>**Transform Feature Module**<br/>Buffer and interpolate position and rotation updates.|**Automatic synchronisation**<br/>Generated code synchronises changes to components for you with a full override control option.|**Harness the power of ECS**<br/>Take full advantage of the Unity Entity Component System (ECS).
+|First look: pre-alpha (from 2018-06-19)|**Build less often**<br/>Run client and server running simultaneously in your Unity Editor.<br/><br/>**Transform Feature Module**<br/>Buffer and interpolate position and rotation updates.|**Automatic synchronisation**<br/>Generated code synchronises changes to components for you with a full override control option.|**Harness the power of ECS**<br/>Tale full advantage of the Unity Entity Component System (ECS).
```
Comment (anne-edwards) [inline docs/Roadmap.md]: Typo - "tale"

File: README.md (hunk 1 above)
Comment (jessicafalk) [reply README.md]: yeah, you need to keep the exact same names for the link

File: docs/Roadmap.md (hunk 3 above)
Comment (jessicafalk) [inline docs/Roadmap.md]: if you look at https://github.com/spatialos/UnityGDK/blob/docs/UTY-484-tidy-up-roadmap/docs/Roadmap.md, the formatting of the "first look" cell doesn't look that nice

File: README.md (hunk 1 above)
Comment (ElleEss) [reply README.md]: I am updating the 'R' to 'r' in this PR which is why it won't work if you view the page from this PR - it points to the existing one.

File: docs/README.md (hunk 2 above)
Comment (ElleEss) [reply docs/README.md]: Am renaming in this PR.

File: docs/Roadmap.md (hunk 3 above)
Comment (ElleEss) [reply docs/Roadmap.md]: Ooh, thanks. Not sure how they crept in there.

Comment (ElleEss) [reply docs/Roadmap.md]: Ooh, thanks. Not sure how they crept in there.

Comment (ElleEss) [reply docs/Roadmap.md]: Found a way to make it work.

File: README.md (hunk 1 above)
Comment (anne-edwards) [reply README.md]: I think you either need to change the file name (Roadmap.md) to lower case in this PR as well, or you need to keep the link as "Roadmap.md" (but obviously the link text can have lower case). Does that make sense?

**Pull request closed without merging**